lib/car.rb

```diff
@@ -0,0 +1,33 @@
+class Car
+  attr_reader :make,
+              :model,
+              :monthly_payment,
+              :loan_length
+              :color
+
+  def initialize(make_model, monthly_payment, loan_length)
+    @make = "Ford"
+    @model = "Mustang"
+    @make_model = make_model
+    @monthly_payment = 1500
+    @loan_length = 36
```

Comment:
We don't want to hard code these values either. We want to set them from the parameters being passed to `initialize`. Similar to what you did with `@make_model = make_model`

```suggestion
    @monthly_payment = monthly_payment
    @loan_length = loan_length
```